cmd/operator.go

```diff
@@ -12,17 +14,27 @@ import (
 	"github.com/libopenstorage/operator/pkg/controller/storagecluster"
 	_ "github.com/libopenstorage/operator/pkg/log"
 	"github.com/libopenstorage/operator/pkg/version"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
 	log "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/rest"
 	"k8s.io/kubernetes/staging/src/k8s.io/sample-controller/pkg/signals"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 const (
-	defaultLockObjectName      = "openstorage-operator"
-	defaultLockObjectNamespace = "kube-system"
-	defaultResyncPeriod        = 30 * time.Second
+	flagLeaderElect              = "leader-elect"
+	flagLeaderElectLockName      = "leader-elect-lock-name"
+	flagLeaderElectLockNamespace = "leader-elect-lock-namespace"
+	flagMetricsPort              = "metrics-port"
+	defaultLockObjectName        = "openstorage-operator"
+	defaultLockObjectNamespace   = "kube-system"
+	defaultResyncPeriod          = 30 * time.Second
+	defaultMetricsPort           = 8999
```

Comment:
Is this to align with 9001 port range ?

Reply:
Yes. But it doesn't really matter. It is used for the operator service. Also it can be changed via an argument.